### PR TITLE
Revamp readme

### LINKED ISSUE #745: Add basic setup instructions to README
e.g: If you're not using the IDE to configure:

* For Posit Connect, first call `addConnectServer()` and then `connectUser()`
* For shinyapps/posit.cloud, call `setAccountInfo()`.

### PR DESCRIPTION
**Pull request by hadley**
Add more details about what you can publish and be a bit more explicit about initial setup. Move tensorflow legal stuff to the place where it's used.

Fixes #745

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -22,8 +22,10 @@ knitr::opts_chunk$set(
 [![R-CMD-check](https://github.com/rstudio/rsconnect/workflows/R-CMD-check/badge.svg)](https://github.com/rstudio/rsconnect/actions)
 <!-- badges: end -->
 
-An R package used for deploying applications to [Posit Connect](https://posit.co/products/enterprise/connect/),
-[ShinyApps.io](https://www.shinyapps.io/), and [Posit Cloud](https://posit.cloud/).
+rsconnect makes it easy to publish your Shiny apps, RMarkdown and quarto documents, and plumbr APIs to [Posit Connect](https://posit.co/products/enterprise/connect/),
```
Comment (aronatkins) [inline README.Rmd:25]: R Markdown and Quarto documents, Plumber APIs

File: README.Rmd [hunk 2]
```diff
@@ -22,8 +22,10 @@ knitr::opts_chunk$set(
...
+[ShinyApps.io](https://www.shinyapps.io/), and [Posit Cloud](https://posit.cloud/) from R. 
+
+(If you're looking for the python equivalent, try [rsconnect-python](https://pypi.org/project/rsconnect-python/))
```
Comment (aronatkins) [inline README.Rmd:28]: Python equivalent.

File: README.Rmd [hunk 3]
```diff
@@ -40,18 +42,16 @@ And the development version from [GitHub](https://github.com/) with:
...
-enabled account on Posit Cloud. Premium, Instructor, and Organization account
-holders can request to participate. See [Posit Cloud's Announcement](https://posit.cloud/learn/whats-new#publishing)
-for more information and to request access.
+* For shinyapps.io, go to your [tokens page](https://www.shinyapps.io/admin/#/tokens) and click "Add Token", then follow the instructions to copy and paste the appropriate call to `setAccountInfo()`. Learn more in the [Getting Started Guide](https://www.shinyapps.io/admin/#/tokens).
 
+* For Posit Connect, first use `addServer()` to register your server with rsconnect, then call either `connectUser()` or `connectApiUser()`. `connectUser()` is a bit simpler if you're in an interactive session; `connectApiUser()` works anywhere but requires a you to copy and paste an API key from your user profile. 
 
-## Legal Stuff
+* For Posit Cloud, login, then click your name in the top-right corner, click "Account", then click "Tokens" in the navbar at the top of the page.  Click "New Token", then "Copy" next to the token you just created, and then follow the instructions to copy and paste the appropriate call to `setAccountInfo()`.
```
Comment (aronatkins) [inline README.Rmd:55]: This drops the "beta" note and link for Posit Cloud; intentional?

**Review by aronatkins — APPROVED** [review]
Typos and posit cloud beta?

File: README.Rmd (hunk 3 above)
Comment (hadley) [reply README.Rmd:55]: It's in GA now: https://posit.cloud/learn/whats-new#publishing-ga

Comment (aronatkins) [reply README.Rmd:55]: Ah. Perfect. Thanks.

**Pull request merged**